<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>130</number>
        </attribute>
        <column>
         <property name="text">
          <string>Username</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>First Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Last Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Department</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Password</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="new_button">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_button">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="first_name_label">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="first_name_lineedit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="last_name_label">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="last_name_lineedit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="email_label">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="email_lineedit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="username_label">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="username_lineedit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="password_label">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="password_edit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="confirm_label">
            <property name="text">
             <string>Confirm</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="confirm_lineedit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="department_label">
            <property name="text">
             <string>Department</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="department_combo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="update_user_button">
            <property name="text">
             <string>Update User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel_button">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="rolelist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="rolecombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_button">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_button">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionnew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionsaveas">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionrecent">
   <property name="text">
    <string>recent</string>
   </property>
  </action>
  <action name="actionlanguage">
   <property name="text">
    <string>language</string>
   </property>
  </action>
  <action name="actionjournal">
   <property name="text">
    <string>journal</string>
   </property>
  </action>
  <action name="actionpassword">
   <property name="text">
    <string>password</string>
   </property>
  </action>
  <action name="actionpasswordreset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>help F1</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionrunPerpetualy">
   <property name="text">
    <string>RunPerpetualy</string>
   </property>
   <property name="iconText">
    <string>RunPerpetualy</string>
   </property>
   <property name="toolTip">
    <string>RunPerpetualy</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="text">
    <string>Share</string>
   </property>
   <property name="toolTip">
    <string>Share</string>
   </property>
  </action>
  <action name="actionAddTask">
   <property name="text">
    <string>AddTask</string>
   </property>
   <property name="toolTip">
    <string>AddTask</string>
   </property>
  </action>
  <action name="actionUnLock">
   <property name="text">
    <string>UnLock</string>
   </property>
   <property name="toolTip">
    <string>UnLock</string>
   </property>
  </action>
  <action name="actionPage">
   <property name="text">
    <string>Page</string>
   </property>
   <property name="toolTip">
    <string>Page</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
